python/build_definitions/openssl.py

```diff
@@ -31,7 +31,7 @@ def build(self, builder: BuilderInterface) -> None:
         if is_mac():
             # On macOS x86_64, OpenSSL 1.0.2 fails to detect the proper architecture.
             configure_cmd = [
-                '/bin/bash', './Configure', 'darwin64-x86_64-cc'] + common_configure_options
```

Comment:
was this not necessary in the first place, or did something change?

Reply:
Because 1.1.1k Configure is written in Perl
```#! /usr/bin/env perl```